src/base.js

```diff
@@ -511,9 +596,15 @@ export default class KintoClientBase {
       throw new Error("A bucket id is required.");
     }
     const path = endpoint("bucket", bucketObj.id);
-    const { last_modified } = { bucketObj };
-    const reqOptions = this._getRequestOptions({ last_modified, ...options });
-    return this.execute(requests.deleteRequest(path, reqOptions));
+    const { last_modified } = { ...bucketObj, ...options };
+    return this.execute(
+      requests.deleteRequest(path, {
```

Comment:
nit: for consistency with other calls, this `path` might have to go next line? or others go on the same line :)